### PR TITLE
Call wrapper function in depthwise_conv unit tests

### PR DESCRIPTION
**Pull request by patriklaurell**

### CODE REVIEW

**Review by mansnils — APPROVED** [review]

File: CMSIS/NN/Tests/UnitTest/requirements.txt [hunk 1]
```diff
@@ -1,81 +1,107 @@
...
-tensorflow-estimator==2.4.0
-termcolor==1.1.0
-typing-extensions==3.7.4.3
-urllib3==1.26.5
+simplegeneric==0.8.1
+simplejson==3.13.2
+six==1.15.0
+soupsieve==1.9.6
+traitlets==4.3.2
+typing==3.7.4.3
```
Comment (freddan80) [inline CMSIS/NN/Tests/UnitTest/requirements.txt:100]: There are a lot of changes in this file. Are all these packages needed? I do miss the tensorflow package.

**Comment by patriklaurell** [comment]
@freddan80 fixed the problem with `requirements.txt`

File: CMSIS/NN/Tests/UnitTest/requirements.txt (hunk 1 above)
Comment (patriklaurell) [reply CMSIS/NN/Tests/UnitTest/requirements.txt]: fixed

File: CMSIS/NN/Tests/UnitTest/README.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ The [Unity test framework](http://www.throwtheswitch.org/unity) is used for runn
 Python3 is required.
 It has been tested with Python 3.6 and it has been tested on Ubuntu 16 and 18.
 
+Make sure to use a `pip` version > 19.0 (or >20.3 for macOS), otherwise tensorflow 2 packages are not available.
```
[the remark this answers is not in the document]
Comment (freddan80) [reply CMSIS/NN/Tests/UnitTest/README.md:12]: As discussed, the TF version will be contained in requirements.txt

**Review by freddan80 — APPROVED** [review]

**Pull request merged**